app/src/Components/Event/EventSummary.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from 'react'
+import styled from 'styled-components/native'
+import EventHeading from './EventHeading'
+import EventDate from './EventDate'
+import EventThumbnail from './EventThumbnail'
+import EventTime from './EventTime'
+import Feather from 'react-native-vector-icons/Feather'
+import { Events } from './types'
```

Comment:
This file doesn't exist. (The project summary originally also pointed here, but the type declarations for the project are in '@/Services/modules/projects/index').